pkg/manager/manager.go

```diff
@@ -58,13 +58,13 @@ func NewManager(config *config.Config) (*Manager, error) {
 		logger:                logrus.WithField("manager", ""),
 	}
 	for _, repoConfig := range config.Repos {
+		if repoConfig["disabled"] == true {
```

Comment:
Repo config is a map[string] interface{}. You need to downcast the value into book before comparison